pandas/_libs/tslib.pyx

```diff
@@ -598,7 +598,8 @@ cpdef array_to_datetime(
         is_same_offsets = len(out_tzoffset_vals) == 1
         if not is_same_offsets:
             raise ValueError(
-                "cannot parse datetimes with mixed time zones unless `utc=True`"
+                "Mixed timezones detected. pass utc=True in to_datetime "
```

Comment:
```suggestion
                "Mixed timezones detected. Pass utc=True in to_datetime "
```

And could you capitalize elsewhere too?

Reply:
Thank you for reviewing this PR. I capitalized the letter 'p'.